resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
It is my personal preference. These are my reasons:
- border-radius is continuous. You see circle changing smoothly every time you resize the element.
- background-image is discrete. Nothing changes unless you cross the 250px threshold. If the user is not resizing the window in the right direction, he might miss the change. Also, are we trying to make the demo phone-friendly?
- circle border that stays circle is something I'd have a very hard time coding in JS today. Seeing RO make it easy makes me a happy developer. Changing background-image based on size feels doable thought some @ rules hack.